core/src/main/java/org/infinispan/configuration/global/AllowListConfigurationBuilder.java

```diff
@@ -11,55 +11,55 @@
 import java.util.stream.Collectors;
 
 import org.infinispan.commons.configuration.Builder;
-import org.infinispan.commons.configuration.ClassWhiteList;
+import org.infinispan.commons.configuration.ClassAllowList;
 import org.infinispan.commons.configuration.attributes.AttributeSet;
 
 /**
- * Configures the {@link org.infinispan.manager.EmbeddedCacheManager} {@link org.infinispan.commons.configuration.ClassWhiteList}.
+ * Configures the {@link org.infinispan.manager.EmbeddedCacheManager} {@link ClassAllowList}.
  *
  * @author Ryan Emerson
  * @since 10.0
  */
-public class WhiteListConfigurationBuilder implements Builder<WhiteListConfiguration> {
```

Comment:
We probably also need to keep this for backwards compatibility in Embedded mode.